README.md

```diff
@@ -62,6 +62,12 @@ cargo +stable t -- --test-threads=1
 ### Run performance tests
 Create a package of IP addresses which are required for performance tests.
 
+You must first fetch the `ips.py` script from the ziggurat-core repository.  Run this:
+
+```bash
+./tools/core_scripts.sh
+```
+
```

Comment:
we could even remove this step and hide this part from the user by calling below `core_scripts.sh` script instead of `ips.py` (make that script a wrapper around `ips.py`):
```
./tools/core_scripts.sh --subnet 1.1.1.0/24 --file src/tools/ips.rs --dev_prefix test_zeth
```

and in the script itself, we would just forward these arguments to the real `ips.py` script call.

check my comment below in the `core_scripts.sh` file

Reply:
For now, I've already moved the one line to the readme.  As for changing the script itself (at this level) I would leave that for Piotr to decide, as this is really new functionality beyond the scope of this PR.  That would be a significant change, including the documentation for the various invocation calls, which would have to be moved into the core_scripts.sh file.  In the future, there may also be other common scripts that we might want to fetch from ziggurat-core.  I prefer orthogonality between the fetching the scripts and the calling of them.

Reply:
Let's put it into improvement bucket with _Possible ToDo_ status.
Maybe there will be some other scripts identified which could have similar approach (call & download in one step).

Reply:
I'd say it's a slippery slope, because you could wind up having duplicated scripts like before (albeit wrappers), which was the **original problem we were fixing**.

Reply:
> original problem we were fixing.

I would say we are discussing two different issues (which can conflict from time to time):

- how to have common scripts in the common external repo and use those as easily as possible
- how to ensure the readme is not clogged with too many instructions _(no user likes massive manuals except for maybe Arch Linux users)_ 😃

the next question is what should

Reply:
I'll leave this comment unresolved but will link it from our improvement bucket, we can discuss this more on Slack (it's not a high priority in any way)